crates/cairo-lint-core/src/fix/helper.rs

```diff
@@ -0,0 +1,127 @@
+//! # Helper Functions for Cairo Lint
+//!
+//! This module provides utility functions to assist in generating fixes for `if-else` conditions
+//! within loops, inverting logical conditions, and processing code blocks.
+//! 
+//! The main tasks of this module include:
+//!
+//! 1. Processing block and `else` clause content, including nested `if-else` constructs.
+//! 2. Inverting logical conditions to their opposite for loop and condition rewriting.
+//! 3. Skipping `break` statements when processing blocks to correctly transform loops.
+//!
+//! These helper functions can be reused in various parts of the Cairo Lint codebase to maintain
+//! consistency and modularity when working with blocks and conditions.
+
+
+use cairo_lang_syntax::node::ast::{BlockOrIf, ElseClause, ExprBlock, Statement};
+use cairo_lang_syntax::node::db::SyntaxGroup;
+use cairo_lang_syntax::node::TypedSyntaxNode;
+
+/// Processes a block of code, formatting its content and ignoring any break statements.
+/// 
+/// # Arguments
+/// 
+/// * `db` - The syntax group which provides access to the syntax tree.
+/// * `block` - The expression block (ExprBlock) to be processed.
+/// * `indent` - A string representing the indentation to be applied to the block's content.
+/// 
+/// # Returns
+/// 
+/// A string representing the formatted content of the block.
```

Comment:
can you add an example of what it does so it's crystal clear ? and same for every function